.github/workflows/auto-approve.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         startsWith(github.event.issue.title, 'publish: getsentry/pytest-sentry@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/relay/py@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/responses@') ||
+        startsWith(github.event.issue.title, 'publish: getsentry/rust-usage-accountant@') ||
```

Comment:
Could you please add the python version as well ? 
https://github.com/getsentry/usage-accountant/

Reply:
done